build.zig

```diff
@@ -5,10 +5,14 @@ pub fn build(b: *std.Build) void {
         .root_source_file = b.path("cpuinfo.zig"),
     });
 
+    const target = b.standardTargetOptions(.{});
```

Comment:
add `optimize` too, for consistency, and for when we get per-module optimization levels :)